keyboards/touchpad/readme.md

```diff
@@ -0,0 +1,15 @@
+# Touchpad
+
+![TouchPad](https://static1.squarespace.com/static/561b7180e4b05a82a1747f0b/59d8985903596eb5953e1803/5a43f205e4966b67c55878a2/1521104083905/IMG_2183.jpg?format=2500w)
+
+A small capacitive 6x6 touchpad for launching programs. [More info](https://jacoburge.co.uk/touch-pad/)
+
+Keyboard Maintainer: QMK Community
```

Comment:
Lines 7 and 8 need two spaces added to the end of each.

Reply:
```suggestion
Keyboard Maintainer: QMK Community  
```

Reply:
Oo nice, this is pretty cool.

Reply:
Isn't it, though? 

Shame it doesn't work for multi-line changes.  Though, it's pretty awesome, regardless.

Also, if you open the comment and hit CTL+G, it will pre-populate the comment with the original line, so you can easily edit it.  
Yay for usability improvements. :)